releasenotes/notes/Fixed-param-gate-target-has-and-get_calibration-d2b9e38d63592b3c.yaml

```
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Fixed an issue with :func:`Target.has_calibration` and :func:`Target.get_calibration` 
```

```suggestion
    Fixed an issue with :meth:`.Target.has_calibration` and :meth:`.Target.get_calibration` 
```
Needs this change for the links to work